next/docs/help.mdx

```diff
@@ -0,0 +1,30 @@
+---
+author: "AnVIL"
+description: "Contact us on the AnVIL Discourse or email our help desk."
+hidden: false
+title: "AnVIL Help desk"
+---
+
+## Getting Help With AnVIL
+
+### AnVIL Support Forum
+
+To ask a question or provide feedback, please feel free to reach out to us on the [AnVIL Support Forum](https://help.anvilproject.org/). If your question is more sensitive, please email our [help desk](mailto:help@lists.anvilproject.org).
```

Comment:
Update these links: `AnVIL Support Forum` and `help desk`  to use the `Link` component.